### PR TITLE
Fix/map button focus

### PR DESCRIPTION
**Pull request by mikkoisa**
- Move zoom buttons to PanControls compoent
- Make UserLocation button aria-hidden. Update button margin adjusting
- Add default outline to links

### CODE REVIEW

File: browserTests/addressTest.js [hunk 1]
```diff
@@ -34,8 +34,8 @@ test('AddressView does render correct view', async (t) => {
   ;
 });
 
-test('AddressView map renders correctly', async (t) => {
-  const zoomOut = await Selector('.leaflet-control-zoom-out');
+test.only('AddressView map renders correctly', async (t) => {
```
Comment (Dremars) [inline browserTests/addressTest.js:37]: I guess this .only was for testing purposes? Should be removed

File: browserTests/browserTest.js [hunk 2]
```diff
@@ -67,8 +67,8 @@ fixture`Map tests`
     await waitForReact();
   });
 
-test('Transit marker visible after zoom', async (t) => {
-  const zoomIn  = Selector('.leaflet-control-zoom-in');
+test.only('Transit marker visible after zoom', async (t) => {
```
Comment (Dremars) [inline browserTests/browserTest.js:70]: I guess this .only was for testing purposes? Should be removed

**Review by Dremars — CHANGES_REQUESTED** [review]

File: browserTests/addressTest.js (hunk 1 above)
Comment (mikkoisa) [reply browserTests/addressTest.js:37]: Good catch! Will remove

**Review by Dremars — APPROVED** [review]
Looks good and this is working amazingly right now! Really like the keyboard usability

**Pull request merged**